### DISCUSSION TITLE
`move_line_up(down)` under `[keys.insert]` inconsistent with `[keys.normal]` movement on a soft-wrapped line.

### DISCUSSION THREAD

**Discussion by AndyJado** [helix-editor/helix#8737] [Troubleshooting] [answered]
normal `jk` works fine on soft-wrapped line.

insert `up/down` treats soft-wrapped lines like it wasn't wrapped.

**Comment by the-mikedavis** [comment, marked as answer]
Use the `move_visual_line_up` and `move_visual_line_down` commands to move within a soft-wrapped line. `move_line_up` and `move_line_down` operate on lines without accounting for soft-wrap. See in the default keymap:

https://github.com/helix-editor/helix/blob/f73e9a8d15fd5a87d472a49808baf42ba403f9bf/helix-term/src/keymap/default.rs#L379-L380

https://github.com/helix-editor/helix/blob/f73e9a8d15fd5a87d472a49808baf42ba403f9bf/helix-term/src/keymap/default.rs#L8-L12
